src/megatron/bridge/recipes/minimax_m2/minimax_m2_5.py

```
@@ -0,0 +1,255 @@
+
+    # Tokenizer
+    cfg.tokenizer.tokenizer_model = _HF_PATH
+
+    # Sequence length
+    seq_length = 2048
+    cfg.model.seq_length = seq_length
+    cfg.dataset.seq_length = seq_length
+
+    # Parallelism
+    # EP must divide 256 (number of experts). EP=32 → 8 experts per rank.
+    # Minimum hardware: 4 nodes × 8 GPUs (TP=1, PP=1, EP=32).
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)